services/galley/src/Galley/API.hs

```diff
@@ -32,44 +25,15 @@ import Network.Wai.Routing hiding (route)
 import Network.Wai.Utilities
 import Network.Wai.Utilities.ZAuth
 import Network.Wai.Utilities.Swagger
-import Network.Wai.Utilities.Server hiding (serverPort)
-import Util.Options
 
-import qualified Control.Concurrent.Async      as Async
 import qualified Data.Predicate                as P
 import qualified Data.Set                      as Set
 import qualified Galley.API.Error              as Error
 import qualified Galley.API.Internal           as Internal
-import qualified Galley.Data                   as Data
 import qualified Galley.Queue                  as Q
 import qualified Galley.Types.Swagger          as Model
 import qualified Galley.Types.Teams.Swagger    as TeamsModel
 import qualified Network.Wai.Predicate         as P
-import qualified Network.Wai.Middleware.Gzip   as GZip
-import qualified Network.Wai.Middleware.Gunzip as GZip
-import qualified System.Logger                 as Log
-
-run :: Opts -> IO ()
```

Comment:
Extracted all of this into `Galley.Run`; running isn't so close to Routes and this file is getting full.